docs/rum_collection/advanced_configuration.md

```diff
@@ -215,15 +296,29 @@ You can use the following methods in `Datadog.Configuration.Builder` when creati
 
 To automatically track views (`UIViewControllers`), use the `.trackUIKitRUMViews()` option when configuring the SDK. By default, views are named with the view controller's class name. To customize it, use `.trackUIKitRUMViews(using: predicate)` and provide your own implementation of the `predicate` which conforms to `UIKitRUMViewsPredicate` protocol:
 
+{{< tabs >}}
+{{% tab "Swift" %}}
 ```swift
 public protocol UIKitRUMViewsPredicate {
     func rumView(for viewController: UIViewController) -> RUMView?
 }
 ```
+{{% /tab %}}
+{{% tab "Objective-C" %}}
+```swift
+@objc
+public protocol DDUIKitRUMViewsPredicate: AnyObject {
+    func rumView(for viewController: UIViewController) -> DDRUMView?
+}
+```
+{{% /tab %}}
```

Comment:
Still Swift 🔍🐞, needs Objective-C.

Reply:
Yes, the thing is that the [objc definition of this protocol is in Swift](https://github.com/DataDog/dd-sdk-ios/blob/master/Sources/DatadogObjc/RUMMonitor%2Bobjc.swift#L49L54), so it might be confusing for the user that looks for this protocol in Xcode. WDYT?

 In objc, this would translate to:
```objective-c
@protocol DDUIKitRUMViewsPredicate

- (DDRUMView * _Nullable)rumViewFor:(UIViewController * _Nonnull)viewController;
@end
```

Reply:
Ah, indeed - I missed this context. Yes, if we only show this interface so devs can conform to it, then presenting original form is best. No value in translating this to Objective-C.